mimic-iv/concepts/measurement/creatinine_baseline.sql

```diff
@@ -1,69 +1,71 @@
--- This query extracts the serum creatinine baselines of adult patients on each hospital admission.
+-- This query extracts the serum creatinine baselines of adult patients
+-- on each hospital admission.
 -- The baseline is determined by the following rules:
---     i. if the lowest creatinine value during this admission is normal (<1.1), then use the value
---     ii. if the patient is diagnosed with chronic kidney disease (CKD), then use the lowest creatinine value during the admission, although it may be rather large.
---     iii. Otherwise, we estimate the baseline using the Simplified MDRD Formula:
+--     i. if the lowest creatinine value during this admission is normal (<1.1),
+--          then use the value
+--     ii. if the patient is diagnosed with chronic kidney disease (CKD),
+--          then use the lowest creatinine value during the admission,
+--          although it may be rather large.
+--     iii. Otherwise, we estimate the baseline using Simplified MDRD:
 --          eGFR = 186 × Scr^(-1.154) × Age^(-0.203) × 0.742Female
---     Let eGFR = 75. Scr = [ 75 / 186 / Age^(-0.203) / (0.742Female) ] ^ (1/-1.154)
```

Comment:
This line looks like it has disappeared. Presumably manually edited?

Reply:
Yeah it was a redundant comment